### DISCUSSION TITLE
OsmAnd and DJI (dev)

### DISCUSSION THREAD

**Discussion by luigifab** [osmandapp/OsmAnd#20474] [Ideas]
Is someone know if it's possible to display my drone when it is flying on OsmAnd?

The answer today is probably no, but is it possible?
My DJI remote controller (with a screen) for my DJI mini pro 3 can connect to WiFi, so my phone and the controller can be connected, but how to exchange GPS position? and how to display it in OsmAnd? (position, altitude, direction)

**Comment by DmitryAlexei** [comment]
OsmAnd can show locations via OsmAnd tracker
 https://osmand.net/docs/user/plugins/osmand-tracker/
